PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -304,43 +439,46 @@ public string ExecuteDebuggingCommandOutNull(string debuggingCommand)
         /// Sets breakpoint for the current runspace.
         /// </summary>
         /// <param name="bp">Breakpoint to set</param>
-        public void SetBreakpoint(PowerShellBreakpoint bp)
+        /// <param name="commandReady">If the runspace is not available, this being true will allow the debugger to issue the set breakpoint as 
+        /// debugger command.</param>
+        public void SetBreakpoint(PowerShellBreakpoint bp, bool commandReady = false)
         {
             IEnumerable<PSObject> breakpoints;
 
             ServiceCommon.Log("Setting breakpoint ...");
             try
             {
+                string file = bp.ScriptFullPath;
+                if (GetDebugScenario() != DebugScenario.Local && _mapLocalToRemote.ContainsKey(bp.ScriptFullPath))

```

Comment:
Agreed. That looks like a typo/bad logic flow on my part. Should have been explicitly for RemoteAttach or RemoteSession. Good catch.